llvm_objects.py

```diff
@@ -162,7 +205,7 @@ def _get_instructions(self, l_block, w_module):
             prev = current_instr
             w_instrs.append(current_instr)
             l_instr = llwrap.LLVMGetNextInstruction(l_instr)
-        return w_instrs
+        return w_instrs[:]
```

Comment:
By doing `self.w_instructions = self._get_instructions(l_block, w_module)` you are assigning to `w_instructions` (which is an immutable field) the list you are creating in `_get_instructions`. The list created in `_get_instructions` is `w_instr`, which gets mutated. Therefore, RPython complains that you are mutating an immutable field (even though the list only gets changed before being assigned to an immutable field).

If I hadn't returned a copy of the list, I would've ended up with this error:
`[translation:ERROR] ListChangeUnallowed: 

resizing list
...`